microcosm_pubsub/sentry.py

```diff
@@ -33,41 +53,80 @@ def before_send(event, hint):
                 if var_name in ["self", "cls"]:
                     continue
                 elif isinstance(value, str):
-                    if value.startswith("<function "):
+                    if value.startswith("<") and value.endswith(">"):
                         continue
                     elif var_name.endswith("_id"):
                         continue
+                    elif value.startswith("http"):
+                        continue
+                elif isinstance(value, dict):
+                    for key in value.keys():
+                        if key.endswith("_id"):
+                            continue
+                        value[key] = "<redacted>"
+                    continue
```

Comment:
can we also let though anything in `{None, [], {}, set(), ""}` - definitely not secret, really helpful for debugging to know that something was unexpectedly empty.